src/components/Admin/TrustedSources.js

```diff
@@ -0,0 +1,55 @@
+import { DataGrid } from './DataGrid';
+import React from 'react';
+import createReactClass from 'create-react-class';
+import Fluxxor from 'fluxxor';
+import constants from '../../actions/constants';
+
+const FluxMixin = Fluxxor.FluxMixin(React);
+const StoreWatchMixin = Fluxxor.StoreWatchMixin("AdminStore");
+
+export const TrustedSources = createReactClass({
+    mixins: [FluxMixin, StoreWatchMixin],
+
+    componentDidMount() {
+      const pipelineKeys = this.getAllPipelineKeys();
+      this.getFlux().actions.ADMIN.load_trusted_sources(pipelineKeys);
+    },
+
+    getAllPipelineKeys() {
+      return constants.DATA_SOURCES.get('all').sourceValues;
+    },
+
+    getStateFromFlux() {
+      return this.getFlux().store("AdminStore").getState();
+    },
+
+    handleSave(rows) {
+      rows.map(row => this.appendRowKey(row));
```

Comment:
the list you're transforming isnt being set to any variable?

Reply:
good catch thanks.